docs/howto_guides/api_gui.md

```diff
@@ -0,0 +1,161 @@
+<br/>
+
+![metadata post json validate](../_static/images/howto_guides/api_gui/metadata_post_json_validate.png)
+Metadata may also be validated in a JSON format, which can be posted in the body of the request and validated against the NMDC schema using the `POST /metatdata/json:validate` endpoint before final submission to the portal.<br/>
+<br/>
+
+![metadata post json validate urls file](../_static/images/howto_guides/api_gui/metadata_post_validate_urls_file.png)
+A text file of urls that point to a JSON object may be supplied using the `POST /metadata/json:validate_urls_file` endpoint. This is helpful for validation of very large JSON metadata or if a user has a link to a JSON file but is not in an environment where it is convenient to download the file and then upload it to validate.<br/>
+<br/>
+
+![metadata get nmdcshema version](../_static/images/howto_guides/api_gui/metadata_get_nmdcschema_version.png)
+To view the [NMDC Schema](https://microbiomedata.github.io/nmdc-schema/) version the database is currently using, try executing the `GET /nmdcschema/version` endpoint<br/>
+<br/>
+
+![metadata get collection stats](../_static/images/howto_guides/api_gui/metadata_get_collection_stats.png)
+To get the NMDC Database collection statistics, like the total count of records in a collection or the size of the collection, try executing the `GET /nmdcschema/collection_stats` endpoint<br/>
+<br/>
+
+![metadata get collection name](../_static/images/howto_guides/api_gui/metadata_get_collection_name.png)
+The `GET /nmdcschema/{collection_name}` endpoint is a general purpose way to retrieve metadata about a specified collection given user-provided filter and projection criteria. Please see the [Collection Names](https://microbiomedata.github.io/nmdc-schema/Database/) that may be retrieved. Please note that only one collection set may be retrieved at a time.<br/>
+<br/>
+
+![metadata get doc_id](../_static/images/howto_guides/api_gui/metadata_get_doc_id.png)
+If the identifier of the record is known, the `GET /nmdcshema/ids/{doc_id}` can be used to retrieve the specified record. Note that only one identifier may be used at a time, and therefore, only one record may be retrieved at a time using this method.<br/>
+<br/>
+
+![metadata get collection_name doc_id](../_static/images/howto_guides/api_gui/metadata_get_collection_name_doc_id.png)
+If both the identifier and the collection name of the desired record is known, the `GET /nmdcschema/{collection_name}/{doc_id}` can be used to retrieve the record. The projection parameter is optionally available for this endpoint to retrieve only desired attributes from a record. Please note that only one record can be retrieved at one time using this method.<br/>
+<br/>
+
+#### Metadata Endpoints Example 1: Get all of the biosamples part of the 1000 Soils Research Campaign Study sampled from Colorado
```

Comment:
I recommend adding "that are" between "biosamples" and "part"

```diff
- Get all of the biosamples part of
+ Get all of the biosamples that are part of
```

Reply:
Added! [f7b1c1d](https://github.com/microbiomedata/NMDC_documentation/pull/27/commits/f7b1c1d4da6321c512d6eb0aff52f06949c7bd55)